civis/io/_tables.py

```diff
@@ -819,24 +858,29 @@ def csv_to_civis(filename, database, table, api_key=None, client=None,
                                   sortkey1=sortkey1, sortkey2=sortkey2,
                                   delimiter=delimiter, headers=headers,
                                   credential_id=credential_id,
+                                  primary_keys=primary_keys,
+                                  last_modified_keys=last_modified_keys,
+                                  escaped=escaped, execution=execution,
                                   polling_interval=polling_interval,
                                   hidden=hidden)
     return fut
 
 
-def civis_file_to_table(file_id, database, table, client=None,
+def civis_file_to_table(file_ids, database, table, client=None,
                         max_errors=None, existing_table_rows="fail",
                         diststyle=None, distkey=None,
                         sortkey1=None, sortkey2=None,
-                        delimiter=",", headers=None,
+                        primary_keys=None, last_modified_keys=None,
+                        escaped=False, execution="immediate",
+                        delimiter=None, headers=None,
                         credential_id=None, polling_interval=None,
                         hidden=True):
     """Upload the contents of a Civis file to a Civis table.
```

Comment:
maybe `one or more Civis files`

Reply:
Would it be worth documenting `CivisImportError`? And maybe documenting under what circumstances errors are thrown when attempting to upload multiple file ids?

Reply:
Good catch - I will update. And yes, we should for sure - I can add a `Raises` section to the docstring here?

Reply:
Oops, not all of these comments showed up while I was reviewing. Adding a `Raises` section is a good idea.

Reply:
> Do the files get joined together (presumably not)?

You are correct; the files do not get concatenated, but rather [entered into a manifest](https://docs.aws.amazon.com/redshift/latest/dg/loading-data-files-using-manifest.html) which is passed to the underlying redshift `COPY` statement. 

> Are they assumed to have exactly the same set of columns in exactly the same order?

Yes - as laid out in the [Helpdesk docs for the new endpoint](https://civis.zendesk.com/hc/en-us/articles/360027016851-Advanced-CSV-Imports-via-the-Civis-API), files should share the same column schema and file format (e.g. same delimiter, encoding, compression, etc.). Some of this can be taken care of by the cleaning step, but users should still take care.

> Are they guaranteed to get appended in the order they're passed in?

I do not believe this is the case, as the load into Redshift happens in parallel.

> If the very last file id fails, do you get an error and no table or an error and a table with all of the files but the last one?

This is addressed below, but the COPY statement is atomic - if a single file fails, the entire load fails, and therefore no table is created. 


The fact that there are so many questions here indicates that this docstring is not nearly sufficient. I will add additional detail to the overall summary as well as the relevant parameters.